### PR TITLE
shanks are no longer invisible

### PR DESCRIPTION
**Pull request by Spacedong**
Fixes issue #4135 and issue #4571
The problem was that the shank was a child of the shard, which randomizes its icon state to be either small, medium, or large in size, referring to a different set of icons than defined by the shank's path. The shank is now classified as /obj/item/weapon/shank instead of /obj/item/shard/shank.
:cl: Spacedong
fix: Fixed shanks being invisible.
/:cl:

### CODE REVIEW

**Comment by zamolxius** [comment]
thank you blutebong :eggplant:

File: hippiestation/code/game/objects/items/ducttape.dm [hunk 1]
```diff
@@ -75,8 +75,8 @@
 		if(replace)
 			user.put_in_hands(new_item)
 		playsound(user, 'hippiestation/sound/misc/ducttape1.ogg', 50, 1)
-	if(istype(W, /obj/item/shard) && !istype(W, /obj/item/shard/shank))
-		var/obj/item/shard/shank/new_item = new(user.loc)
+	if(istype(W, /obj/item/shard) && !istype(W, /obj/item/weapon/shank))
```
Comment (KayeArray) [inline hippiestation/code/game/objects/items/ducttape.dm]: no please don't use the weapon subtype it's supposed to be dead and gone. Use melee instead

Comment (Spacedong) [reply hippiestation/code/game/objects/items/ducttape.dm]: ok, i didnt even know that existed my bad im outdated

**Comment by MoreRobustThanYou** [comment]
whatever, it's a fix, I don't need to wait 5 days for headmemes

**Pull request merged**